app/src/main/java/com/sopt/clody/core/security/login/DefaultLoginSecurityChecker.kt

```
@@ -0,0 +1,61 @@
+            "/system/app/Superuser.apk",
+            "/sbin/su", "/system/bin/su", "/system/xbin/su",
+            "/data/local/xbin/su", "/data/local/bin/su",
+            "/system/sd/xbin/su", "/system/bin/failsafe/su",
+            "/data/local/su",
+        )
+        if (paths.any { File(it).exists() }) return true
+
+        return try {
+            Runtime.getRuntime().exec(arrayOf("/system/xbin/which", "su"))
+                .inputStream.bufferedReader().readLine() != null
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)